src/scrapper/pipelines.py

```diff
@@ -187,3 +187,29 @@ def process_item(self, item, spider):
         if isinstance(item, items.Professor):
             super().process_item(item, spider)
         return item
+
+class CourseGroupPipeline(MySQLPipeline):
+    def __init__(self):
+        MySQLPipeline.__init__(self)
+        self.expected_num = int(self.config['statistics']['num_course_groups'])
+        self.table_name = 'course_group'
+
+    def process_item(self, item, spider):
+        if isinstance(item, items.CourseGroup):
+            print("CG PROCESS")
+            super().process_item(item, spider)
+        return item
+
+
+class CUCGPipeline(MySQLPipeline):
```

Comment:
Can you change the name from `CUCGPipeline` to `CourseUnitGroupPipeline` so it is more perceptible as to what it is and to be more consistent with the naming from the other pipelines?